diff -Nru natow-0.2.10/Makefile natow-0.2.10-new/Makefile
--- natow-0.2.10/Makefile	Wed Nov 14 01:16:59 2001
+++ natow-0.2.10-new/Makefile	Wed Nov 14 01:18:45 2001
@@ -29,7 +29,8 @@
 	mkdir -p $(INSTALLDIR)/$(NATOW)
 # copy natow binary
 	cp natow $(INSTALLDIR)/bin/$(NATOW)
-	cp natow-solo $(INSTALLDIR)/bin/$(NATOWSOLO)
+# NATOW-solo is not ready yet :(
+#	cp natow-solo $(INSTALLDIR)/bin/$(NATOWSOLO)
 # copy data
 	cp -r levels/ $(INSTALLDIR)/$(NATOW)/
 	cp -r images/ $(INSTALLDIR)/$(NATOW)/
